### PR TITLE
toolbar_drawer to toolbar_mode changes

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: _includes/configuration/toolbar-mode.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+## toolbar_mode
+{% if site.productminorversion < 6.1 %}
+<a class="anchor" id="toolbar_drawer"></a>
+> **Note**: The `toolbar_drawer` option was deprecated with the release of {{site.productname}} 5.2. The `toolbar_drawer` option was renamed `toolbar_mode`. This change was made to reflect the range of settings available for this option.
```
Comment (lnewson) [inline _includes/configuration/toolbar-mode.md]: `renamed to`?

Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: either should be fine.

File: _includes/configuration/toolbar-mode.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+{% endif %}
+
+The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
+
+The toolbar modes is not available when using [multiple toolbars]({{site.baseurl}}/configure/editor-appearance/#usingmultipletoolbars) or the [toolbar(n)]({{site.baseurl}}/configure/editor-appearance/#toolbarn) option.
```
Comment (wonderbugtiny) [inline _includes/configuration/toolbar-mode.md]: modes are* not available?

Comment (wonderbugtiny) [inline _includes/configuration/toolbar-mode.md]: modes are* not available?

File: _includes/configuration/toolbar-mode.md (hunk 1 above)
Comment (jhaines) [reply _includes/configuration/toolbar-mode.md]: I prefer "renamed to" as it's more correct to me...

Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: updated

File: _includes/configuration/toolbar-mode.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: fixed

**Pull request merged**